src/ploomber/tasks/notebook.py

```diff
@@ -860,7 +892,7 @@ def __init__(
         params=None,
         ext_in=None,
         static_analysis="regular",
-        local_execution=False,
+        local_execution=True,
```

Comment:
this will break compatibility. can't we keep the old False default and enable this feature?

Reply:
Yeah sure. I just thought it is more intuitive to set it to true.

Reply:
let's revert to the old value to keep backward compatibility